spotbugs-tests/src/test/java/edu/umd/cs/findbugs/ba/Issue547Test.java

```diff
@@ -1,29 +1,24 @@
 package edu.umd.cs.findbugs.ba;
 
-import static edu.umd.cs.findbugs.test.CountMatcher.containsExactly;
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.hamcrest.MatcherAssert.assertThat;
 
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.Iterator;
 import java.util.List;
 
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
 import edu.umd.cs.findbugs.AbstractIntegrationTest;
 import edu.umd.cs.findbugs.SortedBugCollection;
-import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcher;
-import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcherBuilder;
 
 class Issue547Test extends AbstractIntegrationTest {
 
     @Test
     void testLambdas() {
-        performAnalysis("lambdas/Issue547.class");
-        final BugInstanceMatcher bugTypeMatcher = new BugInstanceMatcherBuilder().build();
+        Assertions.assertDoesNotThrow(() -> performAnalysis("lambdas/Issue547.class"));
         SortedBugCollection bugCollection = (SortedBugCollection) getBugCollection();
-        assertThat(bugCollection, containsExactly(0, bugTypeMatcher));
```

Comment:
Original issue (https://github.com/spotbugs/spotbugs/issues/547) is about the analysis throwing exception.